docs/cli/bootstrap/dotfiles/apply.md

```
@@ -0,0 +1,42 @@
+### `-n --dry-run`
+
+Print the actions that would run without writing anything
+
+### `-y --yes`
+
+Skip the confirmation prompt
+
+Examples:
+
+```
+$ mise dotfiles apply
```

markdownlint-cli2 (0.22.1)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)